en/02_Developer_Guides/08_Performance/00_Partial_Caching.md

```diff
@@ -62,57 +61,50 @@ otherwise. By using aggregates, we do that like this:
 %>
 ```
 
-The cache for this will update whenever a page is added, removed or edited.
-
-[note]
-The use of the fully qualified classname is necessary.
-[/note]
-
-[note]
-The use of both `.max('LastEdited')` and `.count()` makes sure we check for any object
-edited or deleted since the cache was last built.
-[/note]
-
-[warning]
-Be careful using aggregates. Remember that the database is usually one of the performance bottlenecks.
-Keep in mind that every key of every cached block is recalculated for every template render, regardless of caching
-result. Aggregating SQL queries are usually produce more load on the database than simple select queries,
```

Comment:
Combined the two notes together and the two warnings together to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.